packages/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy/README.md

```diff
@@ -0,0 +1,46 @@
+# Comunica Wrapper Info Occupancy RDF Resolve Hypermedia Links Queue Actor
+
+[![npm version](https://badge.fury.io/js/%40comunica%2Factor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy.svg)](https://www.npmjs.com/package/@comunica/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy)
+
+A comunica Wrapper Info Occupancy RDF Resolve Hypermedia Links Queue Actor.
+
+This module is part of the [Comunica framework](https://github.com/comunica/comunica),
+and should only be used by [developers that want to build their own query engine](https://comunica.dev/docs/modify/).
+
+[Click here if you just want to query with Comunica](https://comunica.dev/docs/query/).
+
+The purpose of this wrapper is to collect in a file information about the occupancy of the link queue.
+It should not be used in production or in a browser environment.
+To observe non-standard properties of the link queue the `IOptionalLinkQueueParameters` can be extended and the methods can be modified in consequence to process those parameters.
+## Install
+
+```bash
+$ yarn add @comunica/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy
+```
+
+## Configure
+
+After installing, this package can be added to your engine's configuration as follows:
+```text
+{
+  "@context": [
+    ...
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy/^0.0.0/components/context.jsonld"
+  ],
+  "actors": [
+    ...
+    {
+      "@id": "urn:comunica:default:rdf-resolve-hypermedia-links-queue/actors#wrapper-info-occupancy",
+      "@type": "ActorRdfResolveHypermediaLinksQueueWrapperInfoOccupancy"
+    }
+  ]
+}
+```
+**It is preferable that the actor is the last called on the bus by the engine by specifying an `beforeActors` adequately.**
+### Config Parameters
+
+* `filePath`: The file where the information will be save.
+The file name will be extended by an identifier for the query.
+The identifier is either generated by the engine or specify by the user by adding in the
+context the key `@comunica/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupance:query-identifier`
+with an id.
```

Comment:
```suggestion
The identifier is either generated by the engine or specified by the user by setting the key `@comunica/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy:query-identifier`
with an id in the context.
```
Also, where is this query-identifier key specified? Should it not be in the Link Traversal Keys file?

Reply:
Oups legacy of the previous implementation.